test/resources/orders.spec.js

```diff
@@ -130,7 +130,7 @@ describe('ORDERS', () => {
               amount: orderAmount,
               receipt: receipt,
               currency: 'INR',
-              payment_capture: 1,
+              partial_payment: true,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏How is this test passing? normaliseBoolean returns 1 for truthy values. Can you please confirm this?

Reply:
@sidag95 partial_payment is returning 1 
<img width="366" alt="Screenshot 2022-09-13 at 3 12 02 PM" src="https://user-images.githubusercontent.com/89454448/189878085-959e094c-d757-4fdf-9bfd-2d5b178f4919.png">

Reply:
Yes, so why did we change 1 to true then here?

Reply:
@sidag95 because before update. we were send 1 for `true` and 0 for `false` . but many users were getting confused because in documentation its `true` and `false` . so thats we are  using normaliseBoolean here  and without normaliseBoolean . it throwing error 
<img width="673" alt="Screenshot 2022-09-14 at 3 35 59 PM" src="https://user-images.githubusercontent.com/89454448/190127228-1dd23329-5ba5-4923-9f2b-18c2f263ccfc.png">

Reply:
@sidag95 as we discussed this is just a mock and its not hitting any http requests function.